### PR TITLE
fart cooldown variable

### PR DESCRIPTION
**Pull request by SalmonIsTasty**
[help] [qol] [first] [administration] [tested]

## What this does

Adds variable to players to let admins change fart cooldown length

## Why it's good

admin abuse

### CODE REVIEW

**Comment by angryturnip** [comment]
Rename to the original name and I will upvote

**Comment by pedr69** [comment]
u dont need a changelog for stuff the users won't see
and especially not the experimental tag, that's for kanef PR's

**Comment by SalmonIsTasty** [comment]
> u dont need a changelog for stuff the users won't see and especially not the experimental tag, that's for kanef PR's

Okay, changed

**Comment by SalmonIsTasty** [comment]
This is what I get after compiling.

image

Why is this "an undefined var" if it was added in human.dm

**Comment by Inorien** [comment]
> This is what I get after compiling.
> 
> image
> 
> Why is this "an undefined var" if it was added in human.dm

looks like you need to define the var bro

**Comment by SalmonIsTasty** [comment]
After fix, compiled and tested. Works

**Comment by SalmonIsTasty** [comment]
https://github.com/vgstation-coders/vgstation13/assets/56691212/f2b86983-697c-482f-a43a-eafa86d7c498

**Comment by Eneocho** [comment]
Sacrificing all my TC so the shitmin gives me massive stomach issues so I can fart with zero delay.

**Comment by pedr69** [comment]
the newfag babby coder did it
he managed to compile

**Comment by SonixApache** [comment]
i can't believe it worked

**Comment by nervere** [comment]
Good work on your first PR, Salmon!

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	name = "unknown"
 	real_name = "unknown"
 	voice_name = "unknown"
+	var/fartCooldown = 20 SECONDS
```
Comment (Kurfursten) [inline code/modules/mob/living/carbon/human/human.dm]: by convention, new vars typically go with other new vars not in the middle of the inherited vars

just scoot this down a few lines

**Comment by Inorien** [comment]
could we just remove the cooldown altogether

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (SalmonIsTasty) [reply code/modules/mob/living/carbon/human/human.dm]: Moved it to line 13

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -10,6 +10,8 @@
 	var/datum/species/species //Contains icon generation and language information, set during New().
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
 
+    var/fartCooldown = 20 SECONDS
```
Comment (Kurfursten) [inline code/modules/mob/living/carbon/human/human.dm]: instead of a tab/indent, you used spaces here

Comment (SalmonIsTasty) [reply code/modules/mob/living/carbon/human/human.dm]: Yeah I was editing it on the phone. Fixed it now

**Comment by Eneocho** [comment]
Can you make this var (or a similar var) apply to the superfart? Such that it the time it takes to complete a superfart is also affected.

**Comment by Kurfursten** [comment]
Don't make things too hard on him with extra requests for his first PR. Perhaps a good second PR if he likes the idea?

**Comment by pedr69** [comment]
[fetishcontent]

**Pull request merged**